contents/docs/cdp/user-agent-populator.md

```diff
@@ -6,6 +6,8 @@ thumbnail: >-
   https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/cdp/thumbnails/user-agent-enhancer.png
 tags:
   - user-agent
+templateId: 
```

Comment:
I think if we could remove all the github / install stuff that would really help. But that said we can follow up and do that part.